packages/pilot/src/containers/Filter/Toolbar/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { cloneElement } from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  CardActions,
+  Button,
+} from 'former-kit'
+
+import {
+  unless,
+  is,
+  of,
+} from 'ramda'
+
+import ChevronDown32 from 'emblematic-icons/svg/ChevronDown32.svg'
+import ChevronUp32 from 'emblematic-icons/svg/ChevronUp32.svg'
+
+import style from './style.css'
+
+const Toolbar = ({
+  children,
+  collapsed,
+  confirmDisabled,
+  disabled,
+  handleToogleMoreFilters,
+  isClearFilterDisabled,
+  isEmptyOptions,
+  onClear,
+  t,
+}) => {
+  const ensureArray = unless(
+    is(Array),
+    of
+  )
+
+  const renderChildrenInput = (input, index) => (
```

Comment:
Acho que fica mais elegante passarmos os estilos para esta função tbm e isolarmos ela para fora do componente:
```suggestion
  const renderChildrenInput = ({ input, index, style }) => (
```
E como temos mais de 2 parâmetros devemos nomear o que é esperado, recebendo um objeto com os parâmetros necessários.

Reply:
Feito!